### DISCUSSION TITLE
Number of training models

### DISCUSSION THREAD

**Discussion by Dtappledoghuati** [STVIR/pysot#338] [General]
Why do I only get 19 models after training 20 times, for siamrpn_r50_l234_dwxcorr.

**Comment by ZhiyuanChen** [comment]
Sorry, but I am having troubles understanding your issue. Can you provide more details?

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now. 
Please create a new issue should you have other questions.
